### ISSUE TITLE
aria-labels should be added to the list items instead of the media posters

### ISSUE THREAD

**Issue by macjabeth** [macjabeth/showbunny#44]
Having the aria labels on the movie posters is not well-supported. To improve SEO, we should add these aria labels to the list item tags.

This should be done after completing #43.
